csharp/src/Ice/OutgoingRequest.cs

```diff
@@ -207,7 +216,8 @@ public class OutgoingRequestWithEmptyParamList : OutgoingRequest
         private readonly string _operationName;
         private readonly bool _idempotent;
 
-        public OutgoingRequestWithEmptyParamList(string operationName, bool idempotent)
+        public OutgoingRequestWithEmptyParamList(string operationName, bool idempotent, bool oneway) :
```

Comment:
The preferred formatting in C# is `: base(...)` on the new line.

Reply:
Oh, well we're definitely not being consistent about it then... I ripgrepped through for `base(`, and even though we have 3 different ways of writing it, the way I used is the most common!